vite.config.js

```diff
@@ -25,4 +27,5 @@ export default defineConfig({
       },
     }
   },
+  plugins: [dts()],
```

Comment:
This creates the `index.d.ts` file for the exported type information. Without this many folks importing through a bundler with typescript would require not-insignificant amounts of changes and no typing information. It's possible to do it without another package, but it helped simplify the output quite a bit.